### ISSUE TITLE
Python model to javascript model always return same prediction 

### ISSUE THREAD

**Issue by danielreyes9756** [tensorflow/tfjs#5022]
Hi and thanks in advance,

I create my own model in python and I get a 93% of accuracy,
then I convert my model to a js model using the command "tensorflowjs_converter --input_format keras customModel_gs.h5 model/",

I add the model file to my angular project and then I try this:
![imagen](https://user-images.githubusercontent.com/43205522/116851903-3b91b080-abeb-11eb-87b2-8693c24b8fa2.png)

The model charge I can do a summary all is correct except the predition, doesn't care the img a upload, always return the same number 2.

My input_shape is a null,128,128,1 (grayscale), I have 4 classes
![imagen](https://user-images.githubusercontent.com/43205522/116851931-49473600-abeb-11eb-80b1-1b1f307f3f96.png)
![imagen](https://user-images.githubusercontent.com/43205522/116851950-55cb8e80-abeb-11eb-8fb7-34a5f0bb8375.png)

OS Windows 10
issue happens on PC device, im working on angular 11
model crearted on google colaboratory so:
TensorFlow default version and intallation of google colaboratory
Same with Python 3.7.10 default version I didn't install it
Bazel version (I don't know what is Bazel)
Not need CUDA
GPU model and memory, I workerd with the colab gpu acelerator but my own GPU is a GTX 1650 4GB GDDR6

In colab the only command I use where
model.save("route/(customModel_gs.h5")
!pip install tensorflowjs
!tensorflowjs_converter --input_format keras customModel_gs.h5 model/",

**Comment by rthadur** [comment]
@danielreyes9756 is it possible to share a complete codepen example with your converted model ? also can you try below code , thank you 
`let tensor = tf.browser.fromPixels(image, 3)
		.resizeNearestNeighbor([224, 224]) // change the image size
		.expandDims()
		.toFloat()
		.reverse(-1); // RGB -> BGR
	let predictions = await model.predict(tensor).data();
	console.log(predictions);`

**Comment by danielreyes9756** [comment]
this lines doesn't work, the code are the lines of the picture so IDK what to do

**Comment by rthadur** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow.tfjs) since it is not a bug or feature request. There is also a larger community that reads questions there.
